### PR TITLE
Dokumentation Ellipse

### PR DESCRIPTION
**Pull request by lukasb1b**

### CODE REVIEW

File: gdx/src/com/badlogic/gdx/math/Ellipse.java [hunk 1]
```diff
@@ -69,6 +69,10 @@ public Ellipse (Vector2 position, float width, float height) {
 		this.height = height;
 	}
 
+	/** Constructs a new ellipse
+	 * 
+	 * @param position Position vector
+	 * @param size Vektor that determinates the size of the ellipse */
```
Comment (crykn) [inline gdx/src/com/badlogic/gdx/math/Ellipse.java:75]: `Vektor` -> `Vector`

Comment (lukasb1b) [reply gdx/src/com/badlogic/gdx/math/Ellipse.java]: I think I changed it how you want it to be

**Review by crykn — APPROVED** [review]

**Comment by lukasb1b** [comment]
where is the error? I only did documentaton

**Comment by tommyettinger** [comment]
From the build logs:

```
1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':spotlessJavaCheck'.
> The following files had format violations:
      gdx/src/com/badlogic/gdx/math/Ellipse.java
          @@ -72,7 +72,7 @@
           \t/**·Constructs·a·new·ellipse
           \t·*·
           \t·*·@param·position·Position·vector
          -\t·*·@param·size·Size·vector*/
          +\t·*·@param·size·Size·vector·*/
           \tpublic·Ellipse·(Vector2·position,·Vector2·size)·{
           \t\tthis.x·=·position.x;
           \t\tthis.y·=·position.y;
  Run './gradlew :spotlessApply' to fix these violations.
```

As it says, run the spotlessApply task and it will fix this. Then you can commit the fixed files and it should be mergeable.

**Review by crykn — APPROVED** [review]

**Pull request merged**